actions/gha-workflow-validator/jest.config.ts

```diff
@@ -0,0 +1,13 @@
+import {Config} from 'jest'
+export default {
+  displayName: "gha-workflow-validator",
+  preset: "../../jest.preset.js",
+  testEnvironment: "node",
+  transform: {
+    "^.+\\.[tj]s$": "@swc/jest",
+  },
+  moduleFileExtensions: ["ts", "js", "html"],
+  coverageDirectory: "../../coverage/actions/gha-workflow-validator",
+  testPathIgnorePatterns: ["src/__tests__/__fixtures__",  "src/__tests__/__snapshots__/", "src/__tests__/__helpers__"],
+  collectCoverage: true,
```

Comment:
I'd probably put this behind a flag or use the `--coverage` flag instead, its faster to re-run tests without coverage

Reply:
The tests execute in about 0.4 seconds, I think it's ok to leave it enabled and it can be disabled if runtime of tests/coverage inflate.